ProefProeveVeluwe/Assets/Scripts/Runtime/Animal Interaction/Feeding/Feeding.cs

```diff
@@ -33,15 +35,26 @@ private void Update()
 
         // Here we move the object with the food tag to the new position of the finger
         var touch = Input.GetTouch(0);
-        var touchPosition = _camera.ScreenToWorldPoint(touch.position);
-        _hit.transform.position = new Vector3(touchPosition.x,touchPosition.y, _hit.transform.position.z);
+        var touchPosition = MainCamera.ScreenToWorldPoint(touch.position);
+        var position = _hit.transform.position;
+        position = new Vector3(position.x, touchPosition.y, position.z);
+        _hit.transform.position = position;
+        _test = touchPosition.y.ToString();
     }
 
     // When a food object collides with the mouth it will run this script
     private void OnTriggerStay(Collider other)
     {
         //Only the food that has been assigned to the animal will continue the script
         if (other.gameObject != FoodNum[RequiredFoodNum].gameObject) return;
-        HungerBar.value += Time.deltaTime/FeedTime;
+        // HungerBar.value += Time.deltaTime/FeedTime;
+    }
+
+    private void OnGUI()
+    {
+        if (GUI.Button(new Rect(10, 10, 150, 100), _test))
+        {
+            print("F");
```

Comment:
i know it is for debugging but it might be better to put a helpfull message here instead of 'F'  ;)